homeassistant/components/sensor/nsw_fuel_station.py

```diff
@@ -0,0 +1,120 @@
+"""
+Component to display the current fuel prices at a NSW fuel station.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/nsw_fuel_station/
+"""
+import datetime
+from typing import Optional
+import voluptuous as vol
+
+from homeassistant.components.light import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['nsw-fuel-api-client==1.0.7']
+
+CONF_STATION_ID = 'station_id'
+CONF_STATION_NAME = 'station_name'
+CONF_FUEL_TYPES = 'fuel_types'
+CONF_DEFAULT_FUEL_TYPES = ["E10", "U91", "E85", "P95", "P98", "DL",
+                           "PDL", "B20", "LPG", "CNG", "EV"]
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STATION_ID): cv.positive_int,
+    vol.Required(CONF_STATION_NAME): cv.string,
+    vol.Optional(CONF_FUEL_TYPES, default=CONF_DEFAULT_FUEL_TYPES):
```

Comment:
Please limit the default to the most common type. At the moment are 11 sensors created which doesn't make much sense.

Reply:
Will change to ["E10", "U91"]